### PR TITLE
new-session-manager: init at 1.4.0

### PR DESCRIPTION
**Pull request by 6AA4FD**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/new-session-manager/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  hardeningDisable = [ "format" ];
+  mesonFlags = [
+  ];
+
+  src = fetchFromGitHub {
+          owner = "linuxaudio";
+          repo = "new-session-manager";
+          rev = "v${version}";
+          sha256 = "PqOv4tx3NLxL2+GWIUVgL72EQYMyDPIMrAkyby3TZ+0=";
+  };
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/new-session-manager/default.nix:17]:
```suggestion
  src = fetchFromGitHub {
          owner = "linuxaudio";
          repo = "new-session-manager";
          rev = "v${version}";
          sha256 = "PqOv4tx3NLxL2+GWIUVgL72EQYMyDPIMrAkyby3TZ+0=";
  };

  nativeBuildInputs = [ meson pkg-config ninja ];

  buildInputs = [ liblo libjack2 fltk ];

  hardeningDisable = [ "format" ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please squash the package related commits together.

**Comment by 6AA4FD** [comment]
> Please squash the package related commits together.

@SuperSandro2000 done!

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -70,6 +70,12 @@
     githubId = 7414843;
     name = "Nicholas von Klitzing";
   };
+  _6AA4FD = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:73]: Please put this into a separate commit with the message ``maintainers: add _6AA4FD``.

File: pkgs/applications/audio/new-session-manager/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "new-session-manager";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+          owner = "linuxaudio";
+          repo = "new-session-manager";
+          rev = "v${version}";
+          sha256 = "PqOv4tx3NLxL2+GWIUVgL72EQYMyDPIMrAkyby3TZ+0=";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/new-session-manager/default.nix:11]:
```suggestion
    owner = "linuxaudio";
    repo = "new-session-manager";
    rev = "v${version}";
    sha256 = "PqOv4tx3NLxL2+GWIUVgL72EQYMyDPIMrAkyby3TZ+0=";
```

File: pkgs/applications/audio/new-session-manager/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  nativeBuildInputs = [ meson pkg-config ninja ];
+
+  buildInputs = [ liblo libjack2 fltk ];
+
+  hardeningDisable = [ "format" ];
+
+
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/new-session-manager/default.nix:20]:
```suggestion
  hardeningDisable = [ "format" ];

```

**Comment by 6AA4FD** [comment]
@SuperSandro2000 done!

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107997` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>new-session-manager</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

new-session-manager.log:
<details>
`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/new-session-manager/default.nix:24:5:

```
   |
24 |     license = licenses.gpl3;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
</details>

**Comment by 6AA4FD** [comment]
@SuperSandro2000 done!

**Comment by SuperSandro2000** [comment]
@6AA4FD Please fix the merge conflict.

**Comment by 6AA4FD** [comment]
@SuperSandro2000 done!

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107997` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>new-session-manager</li>
</ul>
</details>

**Review by siraben — APPROVED** [review]

**Pull request merged**